Makefile

```diff
@@ -0,0 +1,6 @@
+all:
```

Comment:
what's this file good for?

Reply:
Removed in https://github.com/yarl/vicuna/pull/128